### PR TITLE
CSS Updates

### PR DESCRIPTION
**Pull request by speakerbug**
Updated CSS names to match typical naming conventions and put it in an
external style.css file.

### CODE REVIEW

**Comment by speakerbug** [comment]
I signed it!

File: server-frontend/src/server-frontend/client/style.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+th {
+    font-weight: bold;
+    font-size: larger;
+}
+
+.center {
+    text-align: center;
+}
```
Comment (tadhunt) [inline server-frontend/src/server-frontend/client/style.css:40]: add missing newline

**Review by tadhunt — CHANGES_REQUESTED** [review]
I'm happy to accept the sets of changes from this PR, but please separate the syntax and semantics changes into two PRs. This PR currently does both syntax (moving the CSS into style.css) and semantic (renaming the CSS classes), which are logically separate.

**Comment by speakerbug** [comment]
I just created a new PR for the new naming conventions (#6) and once that's merged in I'll open a new PR to move it into an external stylesheet.

**Pull request closed without merging**